lib/http.js

```diff
@@ -142,16 +142,17 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     validate({query: 'bedrock-profile-http.account'}),
     asyncHandler(async (req, res) => {
       const {account: {id: accountId}} = (req.user || {});
-      const {account} = req.query;
+      const {profileAgentId} = req.params;
+      // this throws NotFoundError
+      const {profileAgent} = await profileAgents.get({id: profileAgentId});
       // FIXME: Use http-sigs and zcaps for
       // TODO: Add permissions to allow access for admin
-      if(account !== accountId) {
+      if(profileAgent.account !== accountId) {
```

Comment:
I think this is an improvement, but I wonder if we won't be able to delete "integrations" any more; I'm not sure if we supported this yet anywhere, though.

Reply:
So you could delete integrations before, but as you probably know by know, the account parameter in the request had to match the authenticated account. We do not seem to delete integrations anywhere at the moment. My idea is find all profileAgents that match the profileAgent your are trying to delete. If the profileAgent has an account, only allow the delete if the account matches the authenticated account. For an integration we could allow it to be deleted if the authenticated account's id is in a profileAgent that is part of the profile, but that would mean that any profileAgent that is a member of a profile could delete an integration that is part of that profile.

Reply:
It looks like we'll be proceeding assuming integrations can't be deleted via this API right now.